skimage/measure/tests/test_moments.py

```diff
@@ -96,3 +100,13 @@ def test_moments_hu():
     hu2 = moments_hu(nu2)
     # central moments must be translation and scale invariant
     assert_almost_equal(hu, hu2, decimal=1)
+
+
+def test_centroid():
+    image = np.zeros((20, 20), dtype=np.double)
+    image[14, 14] = 1
+    image[15, 15] = 1
+    image[14, 15] = 0.5
+    image[15, 14] = 0.5
+    image_centroid = centroid(image)
+    assert_equal(image_centroid, (14.5, 14.5))
```

Comment:
I actually don't love this test because the result is the same with and without weighting. How about:

```python
image[14, 14:16] = 1
image[15, 14:16] = 1/3
image_centroid = centroid(image)
assert_allclose(image_centroid, (14.25, 14.5)
```

Reply:
You'll need to add `from __future__ import division` to the top of the file.

Reply:
@kne42 what do you think of this suggestion for the test though? =)

Reply:
@jni Good point – I’ll change the test to actually account for the weighting.